RecoParticleFlow/PFProducer/python/particleFlow_cfi.py

```diff
@@ -244,3 +273,8 @@
 
 
 
+from Configuration.Eras.Modifier_PF_badHcalMitigation_cff import PF_badHcalMitigation
+PF_badHcalMitigation.toModify(particleFlowTmp,
+        electron_protectionsForBadHcal = dict(enableProtections = cms.bool(True)),
+        photon_protectionsForBadHcal   = dict(enableProtections = cms.bool(True)))
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.